zappa/zappa.py

```diff
@@ -225,6 +225,7 @@ class Zappa(object):
     assume_policy = ASSUME_POLICY
     attach_policy = ATTACH_POLICY
     aws_region = 'us-east-1'
+    log_levels = ['OFF', 'ERROR', 'INFO']
```

Comment:
I'd change this to cloudwatch_log_levels to avoid confusion with Python log levels which we also use